server/src/test/java/org/elasticsearch/index/query/QueryStringQueryBuilderTests.java

```diff
@@ -786,27 +786,36 @@ public void testEnabledPositionIncrements() throws Exception {
     }
 
     public void testToQueryFuzzyQueryAutoFuziness() throws Exception {
-        int length = randomIntBetween(1, 10);
-        StringBuilder queryString = new StringBuilder();
-        for (int i = 0; i < length; i++) {
-            queryString.append("a");
-        }
-        queryString.append("~");
+        for (int i = 0; i < 3; i++) {
+            final int expectedEdits;
+            String queryString;
+            switch (i) {
+                case 0:
+                    queryString = randomAlphaOfLengthBetween(1, 2);
+                    expectedEdits = 0;
+                    break;
+
+                case 1:
+                    queryString = randomAlphaOfLengthBetween(3, 5);
```

Comment:
I was wondering if it worth to use default `DEFAULT_LOW_DISTANCE` and `DEFAULT_HIGH_DISTANCE` but these are package-private